scripts/core/device.yml

```
@@ -395,10 +393,10 @@ params:
       name: hDevice
       desc: |
             [in] handle of the device to partition.
-    - type: "$x_device_partition_property_value_t*"
+    - type: const $x_device_partition_property_t*
       name: Properties
```

this is a pointer argument so should have the `p` prefix

```suggestion
      name: pProperties
```